packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarouselNav/useTeachingPopoverCarouselNav.tsx

```diff
@@ -21,6 +22,34 @@ export const useTeachingPopoverCarouselNav_unstable = (
   const focusableGroupAttr = useFocusableGroup({ tabBehavior: 'limited' });
   const totalPages = useTeachingPopoverCarouselContext_unstable(context => context.totalPages);
   const currentPage = useTeachingPopoverCarouselContext_unstable(context => context.currentPage);
+  const setCurrentPage = useTeachingPopoverCarouselContext_unstable(context => context.setCurrentPage);
+
+  let handleKeyDown = useEventCallback((ev: React.KeyboardEvent<HTMLDivElement>) => {
+    if (ev.isDefaultPrevented()) {
+      return;
+    }
+
+    let target = ev.target as HTMLDivElement;
+    // We only want this enabled at the tablist level, not within
+    if (target.role !== 'tablist') {
```

Comment:
I know this is closed, but if we want tablist-based navigation in the future, focus should never go to the tablist container; only to the tabs within it.